config/application.rb

```diff
@@ -24,6 +24,8 @@ class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
 
+    config.active_model.i18n_customize_full_message = true
```

Comment:
Is this intentional to leave in? are we sure there's are no places where the default translation key is being used directly? (I ran into a few in the old forms)

Reply:
Ah I see, I misread this, I thought it was the setting that would output full translation paths instead of the last key when a translation is missing